sugoi-api-core/src/main/java/fr/insee/sugoi/core/service/impl/OrganizationServiceImpl.java

```diff
@@ -199,8 +199,11 @@ public void delete(String realm, String storageName, String id) {
                     org.addMetadatas(GlobalKeysConfig.USERSTORAGE, us.getName());
                   });
           result.getResults().addAll(temResult.getResults());
-          result.setTotalElements(result.getResults().size());
-          if (result.getTotalElements() >= result.getPageSize()) {
```

Comment:
`result.setSearchToken(temResult.getSearchToken());` is missing compared to UserServiceImpl so you can't continue a request on organization without specifying the storage while you can on User